Sources/Onboarding/OnboardingPreferences.swift

```diff
@@ -84,9 +84,14 @@ extension Preferences {
       key: "defaultBrowserIntro.intro-completed",
       default: false)
     
-    /// Whether system notification showed or not
+    /// Whether system notification scheduled or not
     public static let defaultBrowserNotificationScheduled = Option<Bool>(
       key: "general.default-browser-notification-scheduled",
       default: false)
+    
+    /// Whether a default browser local notification should be shown
+    public static let defaultBrowserNotificationShouldBeShown = Option<Bool>(
```

Comment:
Maybe we can drop showing the NTP banner as well? Or is it handled already. I remember another conversation about the banner some time ago

Reply:
yeah different ticket with bunch of other changes like dont show first week etc

https://github.com/brave/brave-ios/pull/7892